common/djangoapps/util/models.py

```diff
@@ -30,6 +31,21 @@ class Meta(ConfigurationModel.Meta):
         app_label = "util"
 
 
+class LoginRateLimitConfiguration(ConfigurationModel):
```

Comment:
You don't need this to be a config model, you can just have it be a django setting.  With remote-config those changes can be made just as quickly as config models.

Reply:
Will do, thanks!